client/src/pages/Rental/Rent.js

```diff
@@ -0,0 +1,274 @@
+import React, { useState} from "react";
+import Grid from "@material-ui/core/Grid";
+import Container from "@material-ui/core/Container";
+import useStyles from "./RentStyle";
+import TextField from "@material-ui/core/TextField";
+import { Paper, Typography } from "@material-ui/core";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Switch from "@material-ui/core/Switch";
+
+import InputLabel from "@material-ui/core/InputLabel";
+import MenuItem from "@material-ui/core/MenuItem";
+import FormControl from "@material-ui/core/FormControl";
+import Select from "@material-ui/core/Select";
+var classNames = require("classnames");
```

Comment:
Please don't use `var`. Because you're using the `import` syntax, you can do

```javascript
import classNames from 'classnames';
```

Also, quick note on this library. You may want to check out [clsx](https://github.com/lukeed/clsx) instead, since it's the one that is referenced in the [Material UI docs](https://material-ui.com/getting-started/faq/#whats-the-clsx-dependency-for), and it's less than half the size of `classnames`.

Reply:
This is a nice point. I think `classnames` was my suggestion. I had no idea `clsx` is preferred

Reply:
> Please don't use `var`. Because you're using the `import` syntax, you can do
> 
> ```js
> import classNames from 'classnames';
> ```
> 
> Also, quick note on this library. You may want to check out [clsx](https://github.com/lukeed/clsx) instead, since it's the one that is referenced in the [Material UI docs](https://material-ui.com/getting-started/faq/#whats-the-clsx-dependency-for), and it's less than half the size of `classnames`.

Thank you for the comment and the infos john :)